extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/mediaModels/MediaMetadata.java

```diff
@@ -18,16 +18,23 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 import java.io.Serializable;
+import java.util.Date;
 
 /** Metadata about a {@code MediaItem}. */
 public class MediaMetadata implements Serializable {
 
+  @JsonProperty("creationTime")
+  private String creationTime;
```

Comment:
maybe document what this field is, since we're using it for "upload time"?

also should we move the static helper `getUploadTime` here as an instance method, so this mapping (from gphotos concepts "MediaItem" to DTP "upload time" concept) is encapsulated closer to this model?

Reply:
Done. Added comment to explain what the field is. 

I would prefer to keep it the way it is just to follow closely how the models are, if that's okay :)